go.mod

```diff
@@ -3,17 +3,20 @@ module github.com/grafana/cortex-tools
 go 1.16
 
 require (
-	cloud.google.com/go/bigtable v1.2.0
+	cloud.google.com/go/bigtable v1.3.0
```

Comment:
Did you do a general `go get -u`, or are these changes related to the revendoring of Loki?

Reply:
I've only done a `go get github.com/grafana/loki@5b9f5b9efaa52bfa737e963392975e942e83861b` Ah. Should I have done `-u`?

Reply:
Nope, that's good :+1: